pandas/tests/frame/methods/test_replace.py

```diff
@@ -1420,3 +1478,31 @@ def test_replace_period_ignore_float(self):
         result = df.replace(1.0, 0.0)
         expected = pd.DataFrame({"Per": [pd.Period("2020-01")] * 3})
         tm.assert_frame_equal(expected, result)
+
+    def test_replace_value_category_type(self, input_category_df, expected_category_df):
```

Comment:
i would just inline the source and expected frame as you are only using them *inside* this test